modules/manage/pages/iceberg/access-iceberg-topics.adoc

```diff
@@ -0,0 +1,103 @@
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+----
+
+Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name:
+
+[,sql]
+----
+SELECT * FROM streaming.redpanda.<topic-name>;
+----
+
+Spark can use the REST catalog to automatically discover the topic's Iceberg table. Redpanda puts the Iceberg table into a namespace called `redpanda`, creating the namespace if necessary.
+
+Depending on your processing engine, you may need to also create a table in the engine to point the data lakehouse at the table in the catalog. For an example, see xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[].
+
+== Integrate file-system based catalog (`object_storage`)
+
+By default, Iceberg topics use the file-system based catalog (config_ref:iceberg_catalog_type,true,properties/cluster-properties[`iceberg_catalog_type`] cluster configuration set to `object_storage`). Redpanda stores the table metadata in https://iceberg.apache.org/javadoc/1.5.0/org/apache/iceberg/hadoop/HadoopCatalog.html[HadoopCatalog^] format in the same object storage bucket or container as the data files.
+
+If using the `object_storage` catalog type, you provide the object storage URL to an Iceberg client so it can access the catalog and data files for your Redpanda Iceberg tables.
+
+=== Specify metadata location
+
+The config_ref:iceberg_catalog_base_location,true,properties/cluster-properties[`iceberg_catalog_base_location`] property stores the base path for the file-system based catalog if using the `object_storage` catalog type. The default value is `redpanda-iceberg-catalog`. 
+
+CAUTION: Do not change the `iceberg_catalog_base_location` value after you have enabled Iceberg integration for a topic.
+
+If your engine needs the full JSON metadata path, use the following:
+
+```
+redpanda-iceberg-catalog/metadata/redpanda/<topic-name>/v<version-number>.metadata.json
+```
+
+This provides read access to all snapshots in the given version (denoted by `version-number`) of a table. 
+
+NOTE: Redpanda automatically removes expired snapshots on a periodic basis. Snapshot expiry helps maintain a smaller metadata size and reduces the window available for xref:manage:iceberg/query-iceberg-topics.adoc#query-using-time-travel[time travel].  
+
+=== Example file-system based catalog configuration
+
+To configure Apache Spark to use a file system-based catalog, specify at least the following properties:
+
+[,spark]
+----
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = hadoop
+# AWS S3 bucket used for object storage
+spark.sql.catalog.streaming.warehouse = s3a://<bucket-name>/redpanda-iceberg-catalog
+# You may need to configure additional properties based on your object storage provider.
+# See https://spark.apache.org/docs/latest/configuration.html
+----
+
+Depending on your processing engine, you may need to also create a new table to point the data lakehouse at the table location.
```

Comment:
```suggestion
Depending on your processing engine, you may need to also create a new table to point the data lakehouse at the table location.
```
Should this be ...point the data lakehouse to the table location?

Reply:
Updated